pocs/linux/kernelctf/CVE-2023-52447_cos/docs/exploit.md

```diff
@@ -0,0 +1,183 @@
+# Exploit Tech Overview
+
+The vulnerability is that bpf program can hold arraymap pointer without increase refcount if it's from array_of_maps.  
```

Comment:
Could you add the relevant code that's responsible for introducing the bug and explain why it allows you to get a reference to an arraymap without increasing it's refcount?